backend/typescript/middlewares/validators/util.ts

```diff
@@ -77,3 +78,21 @@ export const validateUserEmail = (userEmail: string): boolean => {
     userEmail.split("@")[1] === "uwblueprint.org"
   );
 };
+
+export const validateFormResponses = (
```

Comment:
maybe we can generalize this to a `validateMap` type of function? bc we're changing `formResponses` to map so we could use a generic validator in that case